doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -605,6 +594,9 @@ name "/OF160/"
 
  Beagle's results do not change, no matter how often the user chooses to
  pause and resume the measurements.
+ In fact, the cooling of the CPU during exoneration turns out to be a real
```

Comment:
I tried to avoid "idle", because other processes may run, that don't stress the CPU that much.